dataflow/src/org/checkerframework/dataflow/cfg/CFGBuilder.java

```diff
@@ -4577,6 +4633,8 @@ public Node visitVariable(VariableTree tree, Void p) {
             return node;
         }
 
+        Deque<Object> finalliesLabelsStack = new LinkedList<>();
```

Comment:
Can you move this field to all the other fields in the class?
Can you make the field private or protected, like other fields?